src/type_translator.ts

```diff
@@ -419,6 +419,8 @@ export class TypeTranslator {
         return 'void';
       case ts.TypeFlags.Undefined:
         return 'undefined';
+      case ts.TypeFlags.BigInt:
+        return 'bigint';
```

Comment:
Is it worth naming this something like "bigintPlaceHolder"?

I'm thinking that when Closure does implement support for bigint, landing it might require simultaneously removing our externs definition above.  Where if we instead emitted some slightly-off type, then we can cycle tsickle and closure separately.

Maybe this is easy enough to work around either way though, your call.

Reply:
Done.